### PR TITLE
test(text-field): default test case

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Adding unit tests for default case of Text-Field component

**Solving issue**  
Fixes: [CDEP-3043](https://tegel.atlassian.net/browse/CDEP-3043)

**How to test**  
1. Check default test case of tds-text-field [component](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=4FrlwX&nav=MTVfezA2MzcyM0JBLUEyNkQtNEQyMy1BNUUwLTlFMzJCQ0E1OEQ4OX0)
2. Check if added test look OK
3. Run tests and see if all of them pass

**Checklist before submission**
- [ ] I have added unit tests for my changes (if applicable)
- [x] All existing tests pass
- [ ] I have updated the documentation (if applicable)

**Additional context**  
Other test cases will be added in separate commits. To avoid a long review process and better use short development time I decided to chop it in smaller chunks. 

I have increased pixel tolerance in table tests as usually they fail when 1% of total number of pixel is off different. It tends to happen in components with dropdowns or some kind of change on click etc, here it is filtering and expandable rows...

[CDEP-3043]: https://tegel.atlassian.net/browse/CDEP-3043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by nathalielindqvist — APPROVED** [review]
Tests look good and run as they should, great job! However I have noticed that we have a lot of failing tests for the Table component, both in this PR and in develop. Maybe it's due to the text alignment merge we did yesterday?
image

**Comment by timrombergjakobsson** [comment]
> Tests look good and run as they should, great job! However I have noticed that we have a lot of failing tests for the Table component, both in this PR and in develop. Maybe it's due to the text alignment merge we did yesterday? image

I think those are the ones failing intermittently, I think there is a ticket to fix them?

File: packages/core/src/components/table/table/test/expandable-row/table.e2e.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ test.describe('tds-table-expandable-row', () => {
     await expect(tableComponent).toHaveCount(1);
 
     /* check of diff in component screenshot */
-    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0.01 });
```
Comment (timrombergjakobsson) [inline packages/core/src/components/table/table/test/expandable-row/table.e2e.ts:13]: Why the 0.01 value?

Comment (timrombergjakobsson) [reply packages/core/src/components/table/table/test/expandable-row/table.e2e.ts:13]: Nevermind, I saw your comment now!

Comment (theJohnnyMe) [reply packages/core/src/components/table/table/test/expandable-row/table.e2e.ts:13]: I wrote it under Additional content: 
_I have increased pixel tolerance in table tests as usually they fail when 1% of total number of pixel is off different. It tends to happen in components with dropdowns or some kind of change on click etc, here it is filtering and expandable rows..._

We can check also other ways to improve it, I added tolerance as it is one way to get table tests failing, we can also increase it to higher value too, like 2-5%. There is also `waitFor()` method that can be called items before screenshot is taken, that is another possible improvement too.

**Review by timrombergjakobsson — APPROVED** [review]
Nice!

**Pull request merged**